nominal/core/_multipart.py

```diff
@@ -35,7 +35,12 @@ def _sign_and_upload_part_job(
             "successfully signed multipart upload part",
             extra={"key": key, "part": part, "upload_id": upload_id, "response.url": response.url},
         )
-        put_response = requests.put(response.url, data=data, headers=response.headers)
+        put_response = requests.put(
+            response.url,
+            data=data,
+            headers=response.headers,
+            verify=upload_client._verify,
```

Comment:
Was a bit nervous about the use of an externally defined private variable, but looks like `conjure-python-client` [passes it through as-is to `requests`](https://github.com/palantir/conjure-python-client/blob/develop/conjure_python_client/_http/requests_client.py#L136), so should be OK.